src/it/scala/performanceanalysis/utils/TwitterFutures.scala

```diff
@@ -0,0 +1,24 @@
+package performanceanalysis.utils
+
+import com.twitter.util.{Return, Throw}
+import org.scalatest.concurrent.Futures
+
+/**
+  * Created by Ruud on 13-04-16.
+  */
+trait TwitterFutures extends Futures {
+
+  import scala.language.implicitConversions
```

Comment:
Thanks Tim, but this does not resolve the issue. I still get the timeout when running the test.